modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -257,3 +258,23 @@ function oe_theme_helper_post_update_20014() {
     $block->save();
   }
 }
+
+/**
+ * Create oe_theme_main_content view mode for Iframe media.
+ */
+function oe_theme_helper_post_update_20015() {
+  if (!\Drupal::moduleHandler()->moduleExists('oe_media_iframe')) {
+    // Since core.entity_view_display.media.iframe.oe_theme_main_content is
+    // optional config we have to ensure that module is enabled.
+    return t('Skipping since the oe_media_iframe module is not enabled.');
+  }
+
+  $file_storage = new FileStorage(drupal_get_path('theme', 'oe_theme') . '/config/post_updates/20015_create_view_display_media_iframe');
```

Comment:
It is a little bit tricky when the configs for post updates is located in another module/theme. theoretically, it could lead to naming conflict.

Reply:
Post update hooks don't work in oe_theme. Main config is in oe_theme, so we use this approach.
Discussion is in Slack.

Reply:
Ok for me.